README.md

```diff
@@ -23,21 +23,73 @@
 
 ## Introduction
 
-Jujutsu is a
-[Git-compatible](https://martinvonz.github.io/jj/latest/git-compatibility)
-[DVCS](https://en.wikipedia.org/wiki/Distributed_version_control). It combines
-features from Git (data model,
-[speed](https://github.com/martinvonz/jj/discussions/49)), Mercurial (anonymous
-branching, simple CLI [free from "the
-index"](https://martinvonz.github.io/jj/latest/git-comparison#the-index),
-[revsets](https://martinvonz.github.io/jj/latest/revsets), powerful
-history-rewriting), and Pijul/Darcs ([first-class
-conflicts](https://martinvonz.github.io/jj/latest/conflicts)), with features not
-found in most of them
-([working-copy-as-a-commit](https://martinvonz.github.io/jj/latest/working-copy),
-[undo functionality](https://martinvonz.github.io/jj/latest/operation-log),
-automatic rebase, [safe replication via `rsync`, Dropbox, or distributed file
-system](https://martinvonz.github.io/jj/latest/technical/concurrency)).
+Jujutsu is a powerful [version control system](https://en.wikipedia.org/wiki/Version_control)
+system for software projects. You use it to get a copy of the code, track
+changes to the code, and finally publish those changes for others to see and
+use. Jujutsu is unlike most other systems, because internally it abstracts
+the data model used to store your content from its user facing algorithms and
+interface. This allows it to serve as a version control interface with many
+possible physical backends, such as Git (today), while also open to other
+potential backends, like Mercurial, or even more centralized systems like
+Perforce.
+
+Today, by default, we use the Git data model to store and track content, so
```

Comment:
Do you think it's worth mentioning colocated mode here? That might help readers who would otherwise think the Git interop is only at the exchange level like it is in Sapling, Bazaar (?), hg-git, or git-svn (to svn in that case).

Reply:
It could link to the doc where colocation is explained.
You could mention Breezy (modern version of Bazaar) which interoperates with Git and SVN. (I used only Breezy commands to manage a GitHub repo and the two logs match.)

Reply:
I thought about it, but didn't want to get too technical... Maybe I can reword it.